sdk/logs/src/main/java/io/opentelemetry/sdk/logs/SdkLogger.java

```diff
@@ -7,22 +7,30 @@
 
 import io.opentelemetry.api.logs.LogRecordBuilder;
 import io.opentelemetry.api.logs.Logger;
+import io.opentelemetry.api.logs.LoggerProvider;
 import io.opentelemetry.sdk.common.InstrumentationScopeInfo;
 
 /** SDK implementation of {@link Logger}. */
 final class SdkLogger implements Logger {
 
+  private static final Logger NOOP_LOGGER = LoggerProvider.noop().get("noop");
+
   private final LoggerSharedState loggerSharedState;
   private final InstrumentationScopeInfo instrumentationScopeInfo;
+  private final LoggerConfig loggerConfig;
 
   SdkLogger(
       LoggerSharedState loggerSharedState, InstrumentationScopeInfo instrumentationScopeInfo) {
     this.loggerSharedState = loggerSharedState;
     this.instrumentationScopeInfo = instrumentationScopeInfo;
+    this.loggerConfig = loggerSharedState.getLoggerConfig(instrumentationScopeInfo);
   }
 
   @Override
   public LogRecordBuilder logRecordBuilder() {
+    if (!loggerConfig.isEnabled()) {
```

Comment:
nit: prefer to invert the if

Reply:
Inverting makes it better for this internal caller, but `LoggerConfig#isEnabled` would eventually be a user-facing public API. And naming it `isEnabled()` allows it to align with the default. I.e. if it were named `isDisabled()`, the default would be `false`, which is a double negative.

Reply:
Yeah I was only suggesting to flip the if so that we don't have the `!` negation (internal caller here). I wasn't suggesting making `isDisabled()`.

Reply:
Oh I get what you're saying now. 👍